### PR TITLE
altered the code to use new interfaces/methods from openid4vci client & common module

### PR DESCRIPTION
**Pull request by sksadjad**
changes:
- using the new interfaces imported from newly introduced `openid4vci-common` module
- some changes in naming conventions

### CODE REVIEW

File: src/providers/credential/OpenId4VcIssuanceProvider.ts [hunk 1]
```diff
@@ -150,7 +151,7 @@ class OpenId4VcIssuanceProvider {
 
   public getCredential = async ({credentialType, pin}: IGetCredentialArgs): Promise<CredentialResponse> => {
     const {issuanceOpts} = await this.getServerMetadataAndPerformCryptoMatching();
-
+    console.log(`in OpenId4VcIssuanceProvider.getCredential issuanceOpts: ${JSON.stringify(issuanceOpts)}`)
```
Comment (nklomp) [inline src/providers/credential/OpenId4VcIssuanceProvider.ts:154]:
```suggestion
```

File: src/providers/credential/OpenId4VcIssuanceProvider.ts [hunk 2]
```diff
@@ -140,6 +140,7 @@ class OpenId4VcIssuanceProvider {
         credentialType,
         pin,
       });
+      console.log(`credentialResponses: ${JSON.stringify(credentialResponses)}`)
```
Comment (nklomp) [inline src/providers/credential/OpenId4VcIssuanceProvider.ts:143]:
```suggestion
```

**Review by nklomp — CHANGES_REQUESTED** [review]
Missing package.json update and also some small console.log lines which can go or moved to debug as it works now

File: src/providers/credential/OpenId4VcIssuanceProvider.ts (hunk 2 above)
Comment (sksadjad) [reply src/providers/credential/OpenId4VcIssuanceProvider.ts]: removed that line

File: src/providers/credential/OpenId4VcIssuanceProvider.ts (hunk 1 above)
Comment (sksadjad) [reply src/providers/credential/OpenId4VcIssuanceProvider.ts]: removed that line

**Comment by sksadjad** [comment]
removed the console logs that you mentioned. plus found another place with wrong imports

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**